apps/prairielearn/assets/scripts/questionsTable.js

```diff
@@ -0,0 +1,190 @@
+/* eslint-env browser, jquery */
+/* global _ */
```

Comment:
Rather than using a global, it's better to `import` lodash directly into this file.

Reply:
done